tests/rkt_pid_file_test.go

```diff
@@ -121,14 +121,14 @@ func TestPidFileAbortedStart(t *testing.T) {
 	if err := runChild.SendLine("\035\035\035"); err != nil {
 		t.Fatalf("Failed to terminate the pod: %v", err)
 	}
-	if err := runChild.Wait(); err.Error() != "exit status 1" {
-		t.Fatalf("rkt didn't terminate as expected: %v", err)
+	if err := runChild.Wait(); err == nil || err.Error() != "exit status 1" {
+		t.Fatalf("rkt didn't terminate as expected (expects: \"exit status 1\") got: %v", err)
```

Comment:
I'm not going to mangle to much with it, because there is ongoing work, to have functional test for kvm flavor, and anyway had to rewrite, most of this test. The separation is already there.